.travis.yml

```diff
@@ -111,7 +111,12 @@ before_install:
     else
       brew update
       brew tap homebrew/gui
```

Comment:
This line causes an error message (that doesn't stop the build): https://travis-ci.org/matplotlib/matplotlib/jobs/340786248#L1724

The line can simply be omitted.